.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: node_js
+
+node_js:
+  - 10.16.2
+
+before_script:
+  - rm package-lock.json
+  - npm install
+
+script:
+- npm ci
+- npm test
+- npm pack
+
+after_success:
+# Submit coverage report to https://codecov.io
+- bash <(curl -s https://codecov.io/bash)
+
+# Monitor only these branches
+branches:
+  only:
+  - production
+  - dev
```

Comment:
Nice work @cakhanif.

I've got a few questions though, any reason why only `production` & `dev` branches are monitored?
What happens when a contributor makes multiple contributions to this project? Are they expected to use feature branching or just dev branches?

Reply:
ok, I will make can in all branches